modules/auxiliary/scanner/rdp/cve_2019_0708_bluekeep.rb

```diff
@@ -67,23 +59,23 @@ def run_host(ip)
       print_good(status[1].to_s)
     elsif status == Exploit::CheckCode::Unsupported  # used to display custom msg error
       status = Exploit::CheckCode::Safe
-      print_status("The target service is not running, or refused our connection.")
+      print_status("The target service is not running or refused our connection.")
     else
       print_status(status[1].to_s)
     end
 
     status
   end
 
-  def check_host(ip)
+  def check_host(*)
```

Comment:
Since this is intended to take only one argument, but you do not use it, I would prefer `_ip` to match the use case.

Reply:
Nice, I was torn on that and I like your suggestion.